src/modules/70-pipeline/components/CreatePipelineForm/CreatePipelineForm.tsx

```diff
@@ -60,7 +60,7 @@ export const CreatePipelineForm: React.FC<CreatePipelineFormProps> = props => {
         })
       }}
     >
-      {formikProps => {
+      {(formikProps: any) => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232251)
```suggestion
      {(formikProps: unknown) => {
```